general/administration/hardware-acceleration.md

```diff
@@ -241,18 +245,21 @@ docker run -d \
  jellyfin/jellyfin
 ```
 
-Or with docker-compose >1.28, add the `deploy` section to your Jellyfin service:
+Or with docker-compose >1.28, add the `deploy` and `environment` sections to your Jellyfin service:
 
 ```yaml
 services:
   jellyfin:
     image: jellyfin/jellyfin
-  # ... your Jellyfin config
-  deploy:
-    resources:
-      reservations:
-        devices:
-        - capabilities: [gpu]
+    # ... your Jellyfin config
+    environment:
+      - NVIDIA_DRIVER_CAPABILITIES=all
```

Comment:
Same

Reply:
I find that weird. I could not get it working at all on my local docker-compose no matter what I do.

Should that be reported to the jellyfin repo as a bug?